MODULE.bazel

```diff
@@ -886,3 +891,98 @@ mainnet_versions(
     name = "mainnet_versions",
     path = "//:mainnet-subnet-revisions.json",
 )
+
+# Set up hermetic cc toolchains for binaries and canisters
+toolchains = use_extension("@hermetic_cc_toolchain//toolchain:ext.bzl", "toolchains")
+toolchains.extra_settings(settings = ["//bazel:use_hermetic_cc"])
+use_repo(toolchains, "zig_sdk")
+
+register_toolchains(
+    # if no `--platform` is specified, these toolchains will be used for
+    # (linux,darwin,windows)x(amd64,arm64)
+    "@zig_sdk//toolchain:linux_amd64_gnu.2.28",
+
+    # wasm toolchains
+    "@zig_sdk//toolchain:none_wasm",
+
+    # These toolchains are only registered locally.
+    dev_dependency = True,
+)
+
+# Build libfuzzer to ensure compatibility with our toolchain, and so that it
+# can be optionally linked by the fuzzers.
+http_archive(
+    name = "libfuzzer",
+    build_file_content = """
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "fuzzer",
+    srcs = glob(["*.cpp"]),
+    hdrs = glob(["*.h"]),
+    additional_compiler_inputs = glob(["*.def"]),
+    cxxopts = ["-g", "-O2", "-fno-omit-frame-pointer", "-std=c++17"],
+    linkstatic = True,
+    visibility = ["//visibility:public"],
+)
+""",
+    integrity = "sha256-CLw4JzN3fdo8liWeNzL/lsHfmNBHDE+FsWMnTq5of08=",
+    strip_prefix = "llvm-project-llvmorg-20.1.0/compiler-rt/lib/fuzzer",
+    urls = ["https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-20.1.0.tar.gz"],
+)
+
+# openssl-sys does not build with the vendored feature when using
```

Comment:
out of curiosity, what is the `vendored` feature, and how did this work before?

Reply:
[vendored](https://docs.rs/openssl/latest/openssl/#vendored) would let us push the OpenSSL build to the crate, but it doesn't work directly with `rules_rust`. I pulled this workaround from the linked issue.

Before we were using the [automatic](https://docs.rs/openssl/latest/openssl/#automatic) detection to search through the build environment, but this doesn't work with the hermetic toolchains.

Reply:
gotcha! worth leaving a note

Reply:
Added!